tests/marketaggregator.ts

```diff
@@ -1,7 +1,8 @@
 import { expect } from "chai";
 import { MarketAggregator } from "../src/marketaggregator";
 
-const MAINNET_BETA_ENDPOINT = "https://api.mainnet-beta.solana.com/";
+// TODO: Allow environment variable config of endpoints to avoid rate limits
+const MAINNET_BETA_ENDPOINT = "https://step.rpcpool.com/fe466842f18f30b1c823c2d880df";
```

Comment:
token not secure like this. How hard would it be to get env variables set up on the box?